addons/binding/org.openhab.binding.irtrans/ESH-INF/thing/blaster.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="irtrans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Blaster/Receiver diode connected to an IRtrans transceiver -->
+	<thing-type id="blaster">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="ethernetbridge" />
+		</supported-bridge-type-refs>
+
+		<label>Blaster</label>
+		<description>This is an infrared transmitter that can send infrared commands</description>
+
+		<channels>
+			<channel id="io" typeId="io" />
+		</channels>
+
+		<config-description>
+			<parameter name="led" type="text" required="true">
+				<label>Led</label>
+				<description>The Led on which infrared commands will be emitted</description>
+			</parameter>
```

Comment:
Indeed, that is nicely described in the IRtrans manual ;-) I mean, anyone that buys that IRtrans hardware *has* to go through its manual anyways for the simple reason that is a complicated piece of hardware. Even if you are using their supplied software/server/drivers.